### DISCUSSION TITLE
Rollup Version

### DISCUSSION THREAD

**Discussion by smholsen** [vanilla-extract-css/vanilla-extract#1289] [General]
Hey! 
(cc: @askoufis , @mrm007 )

In our root project where we use vanilla-extract, we have a dev dependency on `rollup@4.x.x`, which results in a peer dependency issue (warn) when we include vanilla-extract. 
```
> @vanilla-extract/rollut-plugin@1.3.0" has incorrect peer dependency "rollup@^2.0.0 || ^3.0.0"
```

I might be wrong, but it seems this stems from the following deps in vanilla-extract: 
https://github.com/vanilla-extract-css/vanilla-extract/blob/master/package.json#L48-L50

I assume it would be desirable to upgrade these to more recent versions;
https://www.npmjs.com/package/rollup-plugin-dts: from 4.x.x to 6.x.x
https://www.npmjs.com/package/rollup-plugin-node-externals: from 5.x.x to 6.x.x
https://www.npmjs.com/package/rollup: from 2.x.x to 4.x.x

Is this something that's already planned for a later major version upgrade of `vanilla-extract`, or is this something we could already submit a PR for?

**Comment by mrm007** [comment]
Thanks for reporting this. The linked `package.json` is for the monorepo, and is unrelated to the dependencies for the Rollup plugin. We use Rollup and those plugins for packaging type definitions: https://github.com/vanilla-extract-css/vanilla-extract/blob/master/scripts/build-dts.ts#L7-L9

The peer dependency for the plugin is declared here: https://github.com/vanilla-extract-css/vanilla-extract/blob/master/packages/rollup-plugin/package.json#L29. If you'd like to contribute a PR, I'll help review it. Just need to add `^4.0.0` to that range, which would be a minor version bump.

**Reply by smholsen** [reply]
Thanks @mrm007 ! 
I've spun up this: https://github.com/vanilla-extract-css/vanilla-extract/pull/1290

**Comment by mrm007** [comment]
Released in [@vanilla-extract/rollup-plugin@1.3.2](https://github.com/vanilla-extract-css/vanilla-extract/releases/tag/%40vanilla-extract%2Frollup-plugin%401.3.2)
